src/sentry/utils/pytest/selenium.py

```diff
@@ -110,38 +127,42 @@ def snapshot(self, name):
         return self
 
     def save_cookie(self, name, value, path='/', expires='Tue, 20 Jun 2025 19:07:44 GMT'):
-        # XXX(dcramer): "hit a url before trying to set cookies"
+        cookie = {
+            'name': name,
+            'value': value,
+            'expires': expires,
+            'path': path,
+            'domain': self.domain,
+        }
+
+        # XXX(dcramer): the cookie store must be initialized via a URL
         if not self._has_initialized_cookie_store:
             logger.info('selenium.initialize-cookies')
             self.get('/')
-            self._has_initialized_cookie_store = True
 
         # XXX(dcramer): PhantomJS does not let us add cookies with the native
         # selenium API because....
         # http://stackoverflow.com/questions/37103621/adding-cookies-working-with-firefox-webdriver-but-not-in-phantomjs
-        # TODO(dcramer): this should be escaped, but idgaf
-        logger.info('selenium.set-cookies.{}'.format(name), extra={
-            'value': value,
-        })
-
-        self.driver.execute_script(
-            "document.cookie = '{name}={value}; path={path}; domain={domain}; expires={expires}';\n".
-            format(
-                name=name,
-                value=value,
-                expires=expires,
-                path=path,
-                domain=self.domain,
+        if isinstance(self.driver, webdriver.PhantomJS):
```

Comment:
Also, shouldn't we delete all code paths (ref phantomjs)?

Reply:
i figured there was no point in removing phantom compat in case we needed it in the future